tinyyaml.lua

```diff
@@ -510,7 +510,7 @@ local function parseseq(line, lines, indent)
     end
     local rest = ssub(line, j+1)
 
-    if sfind(rest, '^[^\'\"%s]*:%s') then
+    if sfind(rest, '^[^\'\"%s]*:$') or sfind(rest, '^[^\'\"%s]*:%s.') then
```

Comment:
Need to add comment for it

Reply:
Comments updated. PTAL~

Reply:
How about trailing whitespace after colon?

Reply:
@spacewander Thanks for pointing it out! Fixed